### PR TITLE
Implementing Jinja2 templates in project

### PR DESCRIPTION
**Pull request by RadekLejba**
https://kindergarten.stxnext.pl/browse/WDP-61

### CODE REVIEW

File: src/presence_analyzer/templates/presence_weekday.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                            loading.hide();
+                            var chart = new google.visualization.PieChart(chart_div[0]);
+                            chart.draw(data, options);
+                        });
+                    }
+                });
+            });
+        })(jQuery);
+    </script>
+{%endblock%}
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/presence_weekday.html]: Empty line on the end of file

File: src/presence_analyzer/templates/presence_weekday.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                    loading.hide();
+                });
+                $('#user_id').change(function(){
+                    var selected_user = $("#user_id").val();
+                    var chart_div = $('#chart_div');
+                    if(selected_user) {
+                        loading.show();
+                        chart_div.hide();
+                        $.getJSON("/api/v1/presence_weekday/"+selected_user, function(result) {
+                            var data = google.visualization.arrayToDataTable(result);
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/presence_weekday.html]: Multiple declaration in js:
```
var aaa = 123,
    bbb = 456,
    ccc = 789;
```

File: src/presence_analyzer/templates/mean_time_weekday.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                            var data = new google.visualization.DataTable();
+                            data.addColumn('string', 'Weekday');
+                            data.addColumn('datetime', 'Mean time (h:m:s)');
+                            data.addRows(result);
+                            var options = {
+                                hAxis: {title: 'Weekday'}
+                            };
+                            var formatter = new google.visualization.DateFormat({pattern: 'HH:mm:ss'});
+                            formatter.format(data, 1);
+
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/mean_time_weekday.html]: Too many empty lines.

File: src/presence_analyzer/templates/base.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            <ul id="navigation">
+                {% for id, caption in navigation_bar %}
+                  <li{% if id == active_page %} id="selected"{% endif
+                  %}><a href="{{id|e}}">{{ caption|e }}</a></li>
+                {% endfor %}
+            </ul>
+            <!--============================-->
+        </div>
+        <div id="content">
+        <h2>test</h2>
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/base.html]: test? it looks like WIP PR

File: src/presence_analyzer/templates/base.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
+ <!--!BASE TEMPLATE FOR WHOLE PROJECT!-->
+
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/base.html]: Too many empty lines.

File: src/presence_analyzer/templates/base.html [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+    <div id="main">
+        <div id="header">
+            <h1>Presence analyzer</h1>
+            <!--!NAVIGATION BAR!-->
+            <!--(file name, Name in nav bar)-->
+            {% set navigation_bar = [
+                ('mean_time_weekday.html', 'Presence by mean'),
+                ('presence_start_end.html', 'Presence by start-end'),
+                ('presence_weekday.html', 'Presence by weekday')
+            ] -%}
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/base.html]: is '-' required here?

File: src/presence_analyzer/templates/base.html [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+<html lang=en>
+<head>
+    <meta charset=utf-8>
+    <title>Presence analyzer</title>
+    <meta name="keywords" content="" />
+    <meta name="description" content=""/>
+    <meta name="author" content="STX Next sp. z o.o."/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <link href="/static/css/normalize.css" media="all" rel="stylesheet" type="text/css" />
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/base.html]: url_for?

File: src/presence_analyzer/templates/mean_time_weekday.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+{% set active_page = "mean_time_weekday.html" %}
+{% block script %}
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+    <script type="text/javascript">
+        google.load("visualization", "1", {packages:["corechart"], 'language': 'pl'});
+    </script>
+    <script type="text/javascript">
+        function parseInterval(value) {
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/mean_time_weekday.html]: you are using this function few times
it is good idea to separete it to any utils file

File: src/presence_analyzer/templates/presence_weekday.html (hunk 2 above)
Comment (RadekLejba) [inline src/presence_analyzer/templates/presence_weekday.html]: This one is acctually in project since i got it.

File:  [hunk 9]
```diff
@@ -0,0 +1,85 @@
+[33mcommit e039fd35c76baf17deed74bb38046c1cf9831e0c[m
```
Comment (RadekLejba) [inline ]: I have no idea how to fix it. Any hints?

File: src/presence_analyzer/templates/presence_weekday.html [hunk 10]
```diff
@@ -1,38 +1,40 @@
...
-                        chart_div.hide();
-                        $.getJSON("/api/v1/presence_weekday/"+selected_user, function(result) {
-                            var data = google.visualization.arrayToDataTable(result);
-                            var options = {};
-                            chart_div.show();
-                            loading.hide();
-                            var chart = new google.visualization.PieChart(chart_div[0]);
-                            chart.draw(data, options);
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+        <script type="text/javascript">
```
Comment (RadekLejba) [inline src/presence_analyzer/templates/presence_weekday.html]: I also added some extra indentation, from here to closing <script> tag.

File: src/presence_analyzer/static/js/mean_time_weekday.js [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+            var selected_user = $("#user_id").val(),
+                chart_div = $('#chart_div');
+            if(selected_user) {
+                loading.show();
+                chart_div.hide();
+                $.getJSON("/api/v1/mean_time_weekday/"+selected_user, function(result) {
+                    $.each(result, function(index, value) {
+                        value[1] = parseInterval(value[1]);
+                    });
+                    console.log(result);
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/mean_time_weekday.js]: Remove please

File: src/presence_analyzer/static/js/presence_weekday.js [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+            loading.hide();
+        });
+        $('#user_id').change(function(){
+            var selected_user = $("#user_id").val(),
+                chart_div = $('#chart_div');
+            if(selected_user) {
+                loading.show();
+                chart_div.hide();
+                $.getJSON("/api/v1/presence_weekday/"+selected_user, function(result) {
+                    var data = google.visualization.arrayToDataTable(result);
```
Comment (dpolcyn) [inline src/presence_analyzer/static/js/presence_weekday.js]: , instead of ;

File: src/presence_analyzer/templates/presence_weekday.html [hunk 13]
```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+{% set active_page = "presence_weekday.html" %}
+{% block script %}
+    <script type="text/javascript" src="https://www.google.com/jsapi"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/presence_weekday.js') }}"></script>
+{%endblock%}
```
Comment (dpolcyn) [inline src/presence_analyzer/templates/presence_weekday.html]: missing spaces

File: src/presence_analyzer/views.py [hunk 14]
```diff
@@ -19,7 +19,15 @@ def mainpage():
...
     Redirects to front page.
     """
-    return redirect('/static/presence_weekday.html')
+    return redirect(url_for('pages', page="presence_start_end.html"))
+
+
+@app.route('/<page>')
+def pages(page):
+    """
+    Renders templates
```
Comment (dpolcyn) [inline src/presence_analyzer/views.py]: Full stop on the end of doc string

**Pull request merged**